README.md

```diff
@@ -1 +1,7 @@
 # Practica de Elixir
```

Comment:
Please improve the README description

Reply:
Ready, Einer. I will work on improving the README description. Thanks